reversestring-SIAS_Opt2/prompts.md

```
@@ -0,0 +1,38 @@
+Create a small web project using two files: **index.html** and **script.js**.
+
+Use the following index.html as the starting seed and modify it only if necessary to add input and result elements:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)